tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "target": "es5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Changing this from es6 to es5 is what makes this a babel replacement step.  It makes the es6 output nearly useless however, so I removed that for the moment.

Reply:
oh, i forgot about this, sorry

it'd be best to emit them from separate paths through compilation.  es5 and es6 end up substantially different in size and speed due to the shims and shams involved, and the es6 downstreams can do a better job of tree shaking when unshimmed